src/main/java/com/redfin/sitemapgenerator/SitemapGenerator.java

```diff
@@ -10,7 +10,7 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.zip.GZIPOutputStream;
-
+import com.sun.deploy.util.StringUtils;

```

Comment:
It is incorrect to add a reference to anything in the com.sun packages. They're private APIs that can change at any time.